app/views/landing-pages/parents/PageParents.vue

```diff
@@ -1145,7 +1144,7 @@ export default {
 .concept-covered-tab {
   display: flex;
   flex-direction: column;
-  justify-content: space-between;
+  justify-content: end;
```

Comment:
[`end`  isn't recognized](https://developer.mozilla.org/en-US/docs/Web/CSS/justify-content#Support_in_Flex_layout) on many browser. Pair it with a `justify-content: flex-end` directly after for maximal browser support.

Reply:
Replaced with `justify-content: flex-end`  thanks.